src/main/java/ch/njol/skript/bukkitutil/ItemUtils.java

```diff
@@ -361,4 +358,36 @@ public static boolean isGlass(Material material) {
 				return false;
 		}
 	}
+
+	/**
+	 * Changes the ItemMeta of the provided ItemStack with the provided Consumer
+	 * @param itemStack ItemStack's ItemMeta to be changed
+	 * @param metaChanger Consumer to change the ItemMeta
+	 * @return ItemStack with the updated ItemMeta
+	 * @param <T>
+	 */
+	public static <T extends ItemMeta> ItemStack changeItemMeta(ItemStack itemStack, Consumer<T> metaChanger) {
+		//noinspection unchecked
+		T itemMeta = (T) itemStack.getItemMeta();
+		metaChanger.accept(itemMeta);
+		itemStack.setItemMeta(itemMeta);
+		return itemStack;
+	}
+
+	/**
+	 * Set's the item back from its original location, see {@link ItemUtils#asItemStack(Object)}
```

Comment:
```suggestion
	 * Updates the provided slot, itemtype or item stack in {@code object} by applying {@code itemStack} to it.
```
use `@see` for the ref to `asItemStack`